calliope/test/test_math.py

```diff
@@ -1,8 +1,8 @@
-import filecmp
 from pathlib import Path
 
 import numpy as np
 import pytest
+from pyomo.repn.tests import lp_diff
```

Comment:
Is there a good reason to replace filecmp with this pyomo functionality?

Reply:
Only that pyomo ignores some immaterial differences between the files (floats and ints are considered equivalent if they are the same (`10` == `10.0`). When there is a diff, it is formatted a bit nicer, too.